pyriemann/clustering.py

```diff
@@ -2,7 +2,11 @@
 import numpy
 from sklearn.base import (BaseEstimator, ClassifierMixin, TransformerMixin,
                           ClusterMixin)
-from sklearn.cluster._kmeans import _init_centroids
+try:                # pragma: no cover
+    from sklearn.cluster._kmeans import _init_centroids
+except ImportError: # pragma: no cover
+    from sklearn.cluster import KMeans
+    _init_centroids = KMeans._init_centroids
```

Comment:
This won't work, see: https://github.com/alexandrebarachant/pyRiemann/issues/92#issuecomment-740641197